go/vt/orchestrator/inst/instance_dao.go

```diff
@@ -351,23 +351,23 @@ func ReadTopologyInstanceBufferable(instanceKey *InstanceKey, bufferWrites bool,
 			waitGroup.Add(1)
 			go func() {
 				defer waitGroup.Done()
-				semiSyncMasterPluginLoaded := false
+				semiSyncPrimaryPluginLoaded := false
 				semiSyncReplicaPluginLoaded := false
 				err := sqlutils.QueryRowsMap(db, "show global variables like 'rpl_semi_sync_%'", func(m sqlutils.RowMap) error {
-					if m.GetString("Variable_name") == "rpl_semi_sync_master_enabled" {
-						instance.SemiSyncMasterEnabled = (m.GetString("Value") == "ON")
-						semiSyncMasterPluginLoaded = true
-					} else if m.GetString("Variable_name") == "rpl_semi_sync_master_timeout" {
-						instance.SemiSyncMasterTimeout = m.GetUint64("Value")
-					} else if m.GetString("Variable_name") == "rpl_semi_sync_master_wait_for_slave_count" {
-						instance.SemiSyncMasterWaitForReplicaCount = m.GetUint("Value")
+					if m.GetString("Variable_name") == "rpl_semi_sync_primary_enabled" {
```

Comment:
:exclamation: :exclamation: :exclamation: This is wrong. MySQL's variables are unaffected by any changes to this PR. This code reads a MySQL variable, and its name is `rpl_semi_sync_master_enabled`

Reply:
Fixed